src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -381,14 +381,14 @@ Copyright (c) .NET Foundation. All rights reserved.
 
       <!-- Copy the deps file if using the build deps file. -->
       <ResolvedFileToPublish Include="$(ProjectDepsFilePath)"
-                             Condition="'$(GenerateDependencyFile)' == 'true' and '$(_UseBuildDependencyFile)' == 'true'">
+                             Condition="'$(GenerateDependencyFile)' == 'true' and '$(_UseBuildDependencyFile)' == 'true' and '$(PublishAot)' != 'true'">
```

Comment:
Do we need this part? `_UseBuildDependencyFile` should be false if we're publishing trimmed:

https://github.com/dotnet/sdk/blob/aa74dca0a303b40dff86939517eac3044847cc35/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets#L829-L837

Are we setting PublishTrimmed too late?

(Do not consider this blocking if this is an ordering issue again - just want to understand why we need to change both places in this file.)

Reply:
Thanks!